packages/paypal-commerce-integration/src/paypal-commerce-ratepay/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -75,6 +78,14 @@ export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStra
             );
         }
 
+        if (!loadingContainerId) {
+            throw new InvalidArgumentError(
```

Comment:
May I ask you to cover this error with test?

Reply:
added